custom/ewsghana/handlers/alerts.py

```diff
@@ -293,8 +293,12 @@ def handle(self):
             text = self.msg.text
 
         if not domain.commtrack_enabled:
-            print "tutaj?"
             return False
+
+        if not verified_contact.sql_location:
```

Comment:
I think it would be better to use `self.sql_location` because it's memoized and it's used later on (so we would only do 1 lookup instead of 2)